app/views/pages/_results.html.erb

```diff
@@ -0,0 +1,16 @@
+<%=@message%>
+<h1>Search Results</h1>
+<h2>Results found:<%= @results_found.count %></h2>
+
+<% @usersearch.hits.each do |hit| %>
```

Comment:
You could use a [partial with collection here](http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections).